segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/TrackedConnection.java

```diff
@@ -73,6 +78,11 @@ public void close() {
         this.connection.close();
     }
 
+    public void close(RunnableWithException callback) {
```

Comment:
1. Please add a Javadoc explaining what this method does and what it does with the argument.
2. What if your argument is null?